.github/workflows/code-coverage.yml

```diff
@@ -1,10 +1,30 @@
 name: Code Coverage
-on: workflow_call
+on:
+  workflow_call:
+    inputs:
+      frontend-path-regexp:
+        required: true
+        type: string
+      backend-path-regexp:
+        required: true
+        type: string
+
 jobs:
   comment-coverage-difference:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Changed files
```

Comment:
Does that mean when we bump the version in the external plugins we will see changed files' coverage instead of the current repo one?

Reply:
Nope, a lot of the new steps won't really change any behaviour in the code coverage check for external plugins (they're so fast to build and test anyway that it doesn't matter) 😊